src/tailwind.css

```diff
@@ -1,8 +1,186 @@
 /* purgecss start ignore */
+
 @tailwind base;
 @tailwind components;
 /* purgecss end ignore */
 
 /* Override utils here*/
 
 @tailwind utilities;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2530;
+}
+
+.joinUsWrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.joinUsWrapper__registerLoginWrapper {
+  display: flex;
+}
+
+.joinUsWrapper__registerLoginWrapper__register {
+  @apply -mx-5;
+  background: linear-gradient(270deg, #ffb421 11.7%, #ff7521 90.81%);
+  border-radius: 9px;
+  border-top-left-radius: 0px;
+  border-bottom-left-radius: 0px;
```

Comment:
and to reset these 2 corners, a class of  `.rounded-l-none` is enough

check https://tailwindcss.com/docs/border-radius/#app